### PR TITLE
Source Facebook: fix formatting and publish new version

### PR DESCRIPTION
**Pull request by keu**
## What
This PR fixes leftovers and failed SAT after #4820 

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

### CODE REVIEW

File: airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml [hunk 1]
```diff
@@ -15,12 +15,12 @@ tests:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
       validate_output_from_all_streams: yes
+      # FB serializes numeric fields as strings
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml:18]: we can do this as a follow up task but we should re-enable schema validation

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml (hunk 1 above)
Comment (keu) [reply airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml:18]: it is already created

**Pull request merged**